app/models/team.rb

```diff
@@ -611,6 +611,66 @@ def get_shorten_outgoing_urls
     self.settings.to_h.with_indifferent_access[:shorten_outgoing_urls] || self.tipline_newsletters.where(content_type: 'rss', enabled: true).exists?
   end
 
+  def get_dashboard_exported_data(filters)
```

Comment:
Suggestion: Add a "type" argument that can be "tipline" or "articles".

Reply:
Done